prow/config/templates/config-ConfigMap.yaml

```diff
@@ -107,7 +106,7 @@ data:
           preset-test-config: "true"
         env:
         - name: TEST_CONFIG_PATH
-.         value: "/etc/config/test_config.yaml"
+          value: "/etc/config/test_config.yaml"
```

Comment:
I guess there is a merge conflict as this was addressed in https://github.com/aws-controllers-k8s/test-infra/pull/262

Reply:
You found that and fixed it before I got to it!  Nicely done